src/components/views/rooms/MessageComposerButtons.tsx

```diff
@@ -250,10 +244,10 @@ class UploadButton extends React.Component<IUploadButtonProps> {
     render() {
         const uploadInputStyle = { display: 'none' };
         return (
-            <AccessibleTooltipButton
+            <CollapsibleButton
                 className="mx_MessageComposer_button mx_MessageComposer_upload"
                 onClick={this.onUploadClick}
-                title={_t('Upload file')}
+                title={_t('Attachment')}
             >
                 <input
```

Comment:
does this input need to be a child of the button? it could just be in the same react fragment, no?

Reply:
Possibly not, I'll try it.

Reply:
Actually, I feel like it's still more correct than before, so I'll leave it unless you have a strong opinion.

Reply:
Fixed in f7b063c375